### ISSUE TITLE
Prisma isn't being set to the latest version during `blitz new`

### ISSUE THREAD

**Issue by flybayer** [blitz-js/blitz#647]
### What is the problem?

During `blitz new` we fetch the latest version for all dependencies and update package.json to match that.

However, this is broken for Prisma.

### Steps to Reproduce

1. `blitz new myapp`
2. Open `myapp/package.json`
3. See this the below which is not the latest version

```
    "@prisma/cli": "2.0.0-beta.3",
    "@prisma/client": "2.0.0-beta.3",
```

### Versions

```
0.13.0
```

### Extra  Info

The issue is this line: https://github.com/blitz-js/blitz/blob/canary/packages/generator/src/utils/fetch-latest-version-for.ts#L22

We can allow an exception for that regex for `@prisma/*`

After you make the change, you can test the cli locally like this:

1. `yarn global remove blitz`
2. Run inside blitz repo root: `yarn link-cli`
3. Run somewhere else outside the blitz repo: `blitz new myapp`

**Comment by josemiguelo** [comment]
Hi @flybayer.

Is the following statement appropriate for the purpose of the fix?

``` javascript
if (!version.match(/\d.x/) && !dep.match(/@prisma\/\w+/)) skipFetch = true
```

**Comment by flybayer** [comment]
@josemiguelo I think so! You can test it to make sure :)

**Comment by josemiguelo** [comment]
I already did so :) Just making sure it does make sense to put both regex on the same statement.
Can I be assigned to this? I'd love to contribute

**Comment by flybayer** [comment]
Ok awesome. And yes for sure! You can submit a PR for this. Let me know if you need any help with that :)

**Comment by flybayer** [comment]
Prisma just released [`2.0.0`](https://github.com/prisma/prisma/releases/tag/2.0.0) which totally eliminates this problem 🤷 :)
